src/test/resources/conf/capabilities/test_caps.json

```diff
@@ -0,0 +1,156 @@
+{
+  "user": "BROWSERSTACK_USERNAME",
+  "key": "BROWSERSTACK_ACCESS_KEY",
+  "application_endpoint": "https://bstackdemo.com",
+  "tests": {
+    "parallel": {
+      "common_caps": {
+        "project": "browserstack-examples",
+        "build": "browserstack-examples-testng",
+        "name": "Parallel Test: ",
+        "browserstack.debug": "true",
+        "browserstack.networkLogs": "true",
+        "browserstack.console": "verbose"
+      },
+      "env_caps": [
```

Comment:
the template file contains only 6 browsers in parallel including mobile browsers.
having more browsers can lead to additional time spent during parallel profile execution for TU course users too and for CEs during customer demos.